### PR TITLE
hosts ui fix

### PR DESCRIPTION
**Pull request by yazhiniloganathan**

### CODE REVIEW

File: app/components/hosts/hosts-detail.html [hunk 1]
```diff
@@ -162,8 +162,8 @@
                 <div class="card-title"> Management Network </div>
                 <div class="col-md-7">
                     <c3chart bindto-id="gauge-kbps-chart">
-                        <chart-column column-id="Mbps" column-values="{{hostdetail.mockHost.managementNetwork.inbound + hostdetail.mockHost.managementNetwork.outbound}}" column-type="gauge" column-color="{{hostdetail.getStatusColor((hostdetail.mockHost.managementNetwork.inbound + hostdetail.mockHost.managementNetwork.outbound)/100*100)}}" />
-                        <chart-gauge min="0" max="100" units="{{hostdetail.mockHost.managementNetwork.inbound + hostdetail.mockHost.managementNetwork.outbound}} Mbps" width="10" />
+                        <chart-column column-id="Mbps" column-values="{{hostdetail.mockHost.management_network.inbound + hostdetail.mockHost.management_network.outbound}}" column-type="gauge" column-color="{{hostdetail.getStatusColor((hostdetail.mockHost.management_network.inbound + hostdetail.mockHost.management_network.outbound)/100*100)}}" />
```
Comment (kmkanagaraj) [inline app/components/hosts/hosts-detail.html]: why do we need to change "managementNetwork" to "management_network"?

**Comment by kmkanagaraj** [comment]
Overall, i see '_' introduced, was this causing the issue?

**Comment by yazhiniloganathan** [comment]
this variable  in mock data provider helper is changed, so need to change
in my files

**Comment by kmkanagaraj** [comment]
I would suggest to change in mock data provider instead changing the controller and html template

File: app/components/clusters/mock-data-provider-helpers.ts [hunk 2]
```diff
@@ -172,6 +172,4 @@ export class MockDataProvider {
         });
         return sort ? _.sortBy(list, key) : list;
     };
-}
```
Comment (kmkanagaraj) [inline app/components/clusters/mock-data-provider-helpers.ts]: This change is not required

**Pull request merged**